server/render/index.js

```diff
@@ -63,6 +63,9 @@ export function renderJsx( view, props ) {
 	  | (_| (_| | | |_| | |_) \\__ \\ (_) |
 	   \\___\\__,_|_|\\__, | .__/|___/\\___/
 	               |___/|_|
+
+	to join the fun, visit: https://automattic.com/work-with-us/
```

Comment:
Some links including references to where the traffic is coming from. Are we using that anywhere or interested in including it here? One example is `?utm_source=houseads&utm_campaign=cw-base64`

Reply:
Since people will have to copy/paste the link, I opted for the simplest possible link – the goal here is more impact than raw volume. We ask applicants during interview how they learned for the position, hopefully they will tell us “the Calypso HTML source” :)